psi4/driver/procrouting/proc.py

```diff
@@ -3862,8 +3833,7 @@ def run_fnodfcc(name, **kwargs):
         ['FNOCC', 'RUN_CEPA'],
         ['FNOCC', 'DF_BASIS_CC'],
         ['SCF', 'DF_BASIS_SCF'],
-        ['SCF', 'DF_INTS_IO'],
-        ['SCF', 'SCF_TYPE'])
```

Comment:
I'm not quite seeing why to separately add SCF_TYPE to the optstash later, just because it's in another fn. As long as it doesn't have to be restored from other fn, harmless to add it here, even if it never gets used.

But this way works, too.

Reply:
I thought it reduced the cognitive load. Anyone writing this code doesn't have to *know* that this random function requires this opt stash state or you leak internal options.

Reply:
Ok. Here’s to a better future of decorators and context managers.